deploy/deployment.yaml

```diff
@@ -75,7 +75,7 @@ spec:
         - name: csm-config-params
           mountPath: /etc/karavi-authorization/csm-config-params
       - name: opa
-        image: docker.io/openpolicyagent/opa
+        image: docker.io/openpolicyagent/opa:0.70.0
```

Comment:
could there be an issue if customer has a different version installed?

Reply:
No there shouldn't be any issues, this change in the script will adjust that image for the user https://github.com/dell/karavi-authorization/pull/357/files#diff-34f23f29df5ca1b9e1605742db999604e1f34d24888793c4615d9cd64870240d.